cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/BuiltInRolesAdministrationTest.scala

```diff
@@ -0,0 +1,329 @@
+package org.neo4j.cypher.docgen
+
+import org.neo4j.cypher.docgen.tooling._
+
+class BuiltInRolesAdministrationTest extends DocumentingTest with QueryStatisticsTestSupport {
+  override def outputPath = "target/docs/dev/ql/administration/security/"
+
+  override def doc: Document = new DocBuilder {
+    doc("Built-in roles", "administration-built-in-roles")
+    database("system")
+    synopsis("This section explains the default privileges of the built-in roles in Neo4j and how to recreate them if needed.")
+
+    p(
+      """All of the commands described in the enclosing section require that the user executing the commands has the rights to do so.
```

Comment:
I'm a bit unsure about if the use of `enclosing` is good here, but I will let some native English speaker like @davidoliverSP2 decide that.